packages/app/src/messages/services/message.service.ts

```diff
@@ -206,6 +206,32 @@ export class MessageService {
     }
   }
 
+  public async deleteMessage(
+    messageId: string
+  ): Promise<RequestResponse<undefined>> {
+    const result = await this.deviceService.request({
+      body: {
+        category: PureMessagesCategory.message,
+        messageID: +messageId,
```

Comment:
🟢 
Just my preferences. Number() is for me more readable than using unary. However, you can skip 😉 
```suggestion
        messageID: Number(messageId),
```